distribution/sdk/ddf-sso-query-widget/src/main/java/org/codice/ddf/ui/Query.java

```diff
@@ -162,7 +165,9 @@ private String createPage(HttpServletRequest request) {
     message.append(" Query result html:\n");
     message.append(sb.toString());
     message.append("\n########################################################################\n");
-    LOGGER.debug(message.toString());
+    if (LOGGER.isDebugEnabled()) {
+      LOGGER.debug(message.toString());
```

Comment:
If the `message` is only being used for debugging, it seems wasteful to always build it up and print it conditionally.  Any reason not to build the message itself conditionally as well (move 163-167 inside the LOGGER.isDebugEnabled() check.

Reply:
Do we want to keep these kind of ascii line breaks? (At least with the thinking that "if everyone did this ..." then our logs would look even worse). 

 We could probably just leave it as 
```java
LOGGER.debug("Query result html:\n {}", sb);
```
without being much worse for wear. (and we wouldn't need to wrap in `isDebugEnabled` since it will short circuit before evaluating arguments. )